tests/console/validate_lvm_with_raid1.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: RAID1 on LVM partition validation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "y2_module_consoletest";
+use testapi;
+use utils;
+use Test::Assert ':all';
+use scheduler 'get_test_data';
+
+sub run {
+    my $self = shift;
+
+    select_console 'root-console';
+
+    $config = get_test_data();
+    $config->{expected_num_devs} = $config->{num_disks};
```

Comment:
here the same, we need back the disks so we need to do the operation to count them in perl that was before. I missed that, sorry.

Reply:
done